tests/e2e/pageObjects/ocisPage.js

```diff
@@ -0,0 +1,23 @@
+const { config } = require('../cucumber.conf')
+
+class OcisPage {
```

Comment:
Also use the same naming style for the class file: `ocisPage.js` -> `OcisPage.js`